libparsec/crates/tests_macros/src/lib.rs

```diff
@@ -1,8 +1,12 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
 use proc_macro::TokenStream;
-use quote::quote;
-use syn::{parse_macro_input, punctuated::Pair, FnArg, ItemFn, LitStr, Pat, Type, TypeReference};
+use proc_macro2::Span;
```

Comment:
Why not proc_macro  (https://doc.rust-lang.org/proc_macro/struct.Span.html) instead of proc_macro2 to stay consistent?